Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,31 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("Enter person " + (i + 1) + " first name:");
+                string fName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + "'s last name:");
+                string lName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s age:");
+                int age = Convert.ToInt32(Console.ReadLine());
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s weight in kg:");
+                float weight = float.Parse(Console.ReadLine());
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s height in cm:");
+                float height = float.Parse(Console.ReadLine());
+
+                Console.WriteLine(fName + " " + lName + "is " + age + " years old, their weight is " + weight + " kg and their height is " + height + " cm.");
```

Comment:
Especially here making use of interpolated strings is gold.
If you really need to concatenate strings that often you should make use of a StringBuilder.
This is not needed here and a bit out of the lessons scope, so just use the method I showed you. :)

Reply:
Would it be ok if we hold of on this until it is covered in the lessons? I don't fully understand what you are trying to teach me right now. Maybe if it was in context with the lesson material and I had Kai explain it in a video I would be able to grasp the concept better. I would feel more comfortable approaching it when it is inside scope.

Reply:
Yes, totally! Sorry. I thought this was told already.
However, keep it in mind and feel free to ask me here or in Discord (EarLington) if you want me to tell you more about this! :)

Reply:
Sounds great. Thank you. I have to read the documentation about string interpolation. It's not a concept I am familiar with yet. Thank you again for helping me with my PR.

Reply:
I am glad when I can help! :)
Please continue telling me whenever something is beyond your current scope or when you do not understand something I am telling you. 👍